native/CMakeLists.txt

```diff
@@ -606,6 +662,8 @@ if(ANDROID)
         cocos/platform/android/modules/System.cpp
         cocos/platform/android/modules/System.h
     )
+elseif(OPENHARMONY)
+    # Because there will be a definition of OHOS in harmonyos, it is necessary to define the macro of OPENHARMONY.
```

Comment:
So, OHOS is used for HarmonyOS?

Reply:
Yes, OHOS is defined in both openharmony and HarmonyOS